src/module-elasticsuite-tracker/Model/Healthcheck/TrackerPendingEvents.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\Elasticsuite
+ * @author    Pierre Gauthier <pigau@smile.fr>
+ * @copyright 2025 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteTracker\Model\Healthcheck;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Notification\MessageInterface;
+use Magento\Framework\UrlInterface;
+use Smile\ElasticsuiteAnalytics\Helper\Data as AnalyticsHelper;
+use Smile\ElasticsuiteCore\Api\Healthcheck\CheckInterface;
+use Smile\ElasticsuiteCore\Model\Healthcheck\AbstractCheck;
+use Smile\ElasticsuiteTracker\Api\EventQueueInterface;
+
+/**
+ * Elasticsuite tracker pending events check.
+ * Checks if the Elasticsuite tracker has pending events in the event queue,
+ * which may indicate that the cron is not running.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteTracker
+ */
+class TrackerPendingEvents extends AbstractCheck
+{
+    /**
+     * Events queue.
+     * @var EventQueueInterface
+     */
+    private EventQueueInterface $eventQueue;
+
+    /**
+     * Analytics config helper.
+     * @var AnalyticsHelper
+     */
+    private AnalyticsHelper $config;
+
+    /**
+     * Constructor.
+     *
+     * @param EventQueueInterface $eventQueue Events queue.
+     * @param AnalyticsHelper     $config     Analytics config helper.
+     * @param UrlInterface        $urlBuilder URL builder.
+     * @param int                 $sortOrder  Sort order (default: 50).
+     * @param int                 $severity   Severity level.
+     */
+    public function __construct(
+        EventQueueInterface $eventQueue,
+        AnalyticsHelper $config,
+        UrlInterface $urlBuilder,
+        int $sortOrder = 50,
+        int $severity = MessageInterface::SEVERITY_MINOR
+    ) {
+        parent::__construct($urlBuilder, $sortOrder, $severity);
+        $this->eventQueue = $eventQueue;
+        $this->config = $config;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getIdentifier(): string
+    {
+        return 'tracker_pending_events';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getStatus(): string
+    {
+        return ($this->getPendingEventsCount() ? CheckInterface::STATUS_FAILED : CheckInterface::STATUS_PASSED);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDescription(): string
+    {
+        $description = __('The tracker events seem to be frequently indexed as expected.');
```

Comment:
Please replace "frequently" by "regularly". It does not change your target translations for FR, DE and NL.